ci_support/run_docker_build.sh

```diff
@@ -39,9 +38,8 @@ echo "$config" > ~/.condarc
 # A lock sometimes occurs with incomplete builds. The lock file is stored in build_artefacts.
 conda clean --lock
 
-conda update --yes --all
-conda install --yes conda-build
-conda info
+conda install --yes --quiet conda-forge-build-setup
```

Comment:
So on Linux things may pass as long as `conda-forge-build-setup` is properly pinned.

(That is why I no longer use `conda-forge-build-setup` locally. It hides the build tool versions and makes it harder for the end user to know what is going on.)